modules/consensus/accept.go

```diff
@@ -24,7 +24,7 @@ var (
 )
 
 // validHeader does some early, low computation verification on the block.
-func (cs *ConsensusSet) validHeader(tx *bolt.Tx, b types.Block) error {
+func (cs *ConsensusSet) validHeader(tx dbTx, b types.Block) error {
```

Comment:
dbTx strikes me as a weird name... but I guess it's fine. It's on the edge, but it's pretty obvious that dbTx stands for databaseTransaction... I think?